lib/modules/manager/pipenv/extract.ts

```diff
@@ -111,6 +105,19 @@ function extractFromSection(
   return deps;
 }
 
+function isPipRequirements(
+  section?:
+    | Record<string, PipRequirement>
+    | Record<string, string>
+    | PipSource[],
+): section is Record<string, PipRequirement> {
+  return (
+    !is.array(section) &&
+    is.object(section) &&
+    !Object.values(section).find((dep) => !is.object(dep) && !is.string(dep))
```

Comment:
```suggestion
    !Object.values(section).some((dep) => !is.object(dep) && !is.string(dep))
```
some returns boolean

Reply:
Thanks, I didn't know about that one!